src/main/resources/swagger/agora.yaml

```diff
@@ -285,6 +318,38 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+    delete:
+      summary: Redact a single configuration.
+      description: |
+        Redacts a configuration and all of its associated configurations
+      parameters:
+        - name: namespace
+          in: path
+          description: Namespace of config.
+          required: true
+          type: string
+        - name: name
+          in: path
+          description: Name of config.
+          required: true
+          type: string
+        - name: snapshotId
+          in: path
+          description: Snapshot ID of config.
+          required: true
+          type: number
+      tags:
+        - Configurations
+      responses:
+        200:
+          description: |
+            An integer showing how many configurations were deleted. Should be 1.
+          schema:
+            type: number
+        default:
+          description: Error upon redaction
+          schema:
+            $ref: '#/definitions/Error'
   /methods/{namespace}/permissions:
```

Comment:
All of the permissions methods are tagged as NamespacePermissions and Permissions respectively. So everything for /methods and /configurations end up in the same frame. Do you like this distinction? If it were me I might tag everything Methods or Configurations. So then things would be grouped by what kind of entity they refer to. Up to you